### PR TITLE
maxima: Update to 5.45.1

### PR DESCRIPTION
**Pull request by judaew**
###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 11.5 20G71 x86_64
Xcode 12.5.1 12E507

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [x] checked your Portfile with `port lint`?
- [x] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: math/maxima/Portfile [hunk 1]
```diff
@@ -23,7 +23,7 @@ long_description \
         facilities via rlwrap. \
         * \"xmaxima\" is a graphical interface for maxima, written in Tcl/Tk.
 
-homepage        http://maxima.sourceforge.net/
+homepage        https://maxima.sourceforge.net/
```
[the remark this answers is not in the document]
Comment (judaew) [reply math/maxima/Portfile]: Thanks, done

**Comment by reneeotten** [comment]
@judaew can you please fix the conflict? I know you didn't change the `maxima-devel` port, but did you (or can you) look why that one is failing on the CI?

**Comment by judaew** [comment]
I tried updating maxima-devel but that failed. It looks like this version cannot be built with texinfo version 6.8 or higher. See https://sourceforge.net/p/maxima/bugs/3744/

@reneeotten what is the best way to proceed in this case?

**Comment by reneeotten** [comment]
> I tried updating maxima-devel but that failed. It looks like this version cannot be built with texinfo version 6.8 or higher. See https://sourceforge.net/p/maxima/bugs/3744/
> 
> @reneeotten what is the best way to proceed in this case?

I guess it's not too surprising that a `-devel` port will sometimes fail to build... I would say we can just wait for upstream to fix that issue. Updating the `maxima` port to the latest stable release is a good idea anyway and that port seems to build fine. 

I would suggest to fix the conflict (i.e., the `revision` field was updated because of the update of another port)  and then this can be merged.

**Comment by judaew** [comment]
Done.

I checked this port using `port install -vst maxima`, but in CI seems it's also built correctly.

**Pull request merged**